src/main/java/com/cascadebot/cascadebot/ShutdownHandler.java

```diff
@@ -29,6 +33,7 @@ public static void exitWithError() {
     }
 
     private static void shutdown() {
+        logger.info("Shutting down!");
```

Comment:
We should probably say more then just shutting down here. this is the graceful shutdown so maybe say like `Bot shuting down gracefully` or something along those line. @binaryoverload opinions?

Reply:
I've changed it to "Bot shutting down gracefully" for now, although if Binary wants to add any further opinions on that later

Reply:
Add a `!` we like to have enthusiasm 😉  But yeah that's fine